.changeset/odd-avocados-care.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-node': patch
```

Comment:
```suggestion
'@backstage/plugin-auth-node': minor
```

This changeset will have to be reworded now with a **BREAKING:** section mentioning that `getIdentity` will throw an `AuthenticationError` if `getIdentity` is called without a valid authorization header.